indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -38,12 +38,23 @@ CategoryStatistics.propTypes = {
 };
 
 function getGraphData(data) {
-  const serialize = statsData =>
-    Object.entries(statsData)
-      .filter(([year]) => year <= data.maxYear + 3 && year >= data.minYear)
-      .map(([year, number]) => [moment(year, 'YYYY'), number]);
+  const serialize = statsData => {
+    const maxDate = parseInt(
+      Object.keys(statsData)
+        .reverse()
+        .find(date => date <= data.maxYear + 3),
+      10
+    );
+    return [
+      [moment(data.minYear - 1, 'YYYY'), null],
+      ...Object.entries(statsData)
+        .filter(([year]) => year <= data.maxYear + 3 && year >= data.minYear)
```

Comment:
`year <= maxDate`? Same functionality, but only one place where we use the year+3 logic